<dec f='include/boost/property_tree/exceptions.hpp' l='34' type='void boost::property_tree::ptree_error::~ptree_error()'/>
<inh f='include/c++/4.9/stdexcept' l='121' c='_ZNSt13runtime_errorD1Ev'/>
<def f='include/boost/property_tree/detail/exception_implementation.hpp' l='38' type='void boost::property_tree::ptree_error::~ptree_error()'/>
<ovr f='include/boost/property_tree/detail/exception_implementation.hpp' l='51' c='_ZN5boost13property_tree14ptree_bad_dataD1Ev'/>
<ovr f='include/boost/property_tree/detail/exception_implementation.hpp' l='71' c='_ZN5boost13property_tree14ptree_bad_pathD1Ev'/>
<ovr f='include/boost/property_tree/detail/file_parser_error.hpp' l='37' c='_ZN5boost13property_tree17file_parser_errorD1Ev'/>
